development/cbmc/README

```diff
@@ -0,0 +1,118 @@
+CBMC
+
```

Comment:
This readme seems out of control

Reply:
@aclemons  How so?

Reply:
The README is not meant to be a copy of the website or full intro to a project, just a short description of the thing being packaged and any notes on how to use the script or run the resulting package. It definitely does not need a table with all the details about support for Windows, Macos and other linux distributions. 

https://slackbuilds.org/templates/README

Reply:
@aclemons  oh I see. thanks